commit 0d090028ca55581ac783176a09cc9be3082dd47f
Author: Yuriy Syrota <ysyrota@gmail.com>
Date:   Fri Jul 6 15:57:32 2012 +0300

    Refixed bug #1473 (When we set XmNvisibleItemCount resource available in XmComboBox, the widget displays the wrong number of items)

#diff --git a/ChangeLog b/ChangeLog
#index 1614cfe..7e96f48 100644
#--- a/ChangeLog
#+++ b/ChangeLog
#@@ -1,3 +1,8 @@
#+2010-08-19  Ihor Hayvuk <ihayvuk@ics.com>
#+	* lib/Xm/ComboBox.c:
#+	Refixed bug #1473 (When we set XmNvisibleItemCount resource available in 
#+	XmComboBox, the widget displays the wrong number of items)
#+
# 2010-08-18  Ihor Hayvuk <ihayvuk@ics.com>
# 	* lib/Xm/LabelG.c:
# 	Refixed bug #1517 (LabelGadget draws over parent window with the background color)
diff --git a/lib/Xm/ComboBox.c b/lib/Xm/ComboBox.c
index 0f5a809..ec0acf9 100644
--- a/lib/Xm/ComboBox.c
+++ b/lib/Xm/ComboBox.c
@@ -587,7 +587,7 @@ Initialize(Widget    request,	/* unused */
   child_args = (ArgList) XtMalloc(sizeof(Arg) * *num_args);
   for (i = 0; i < *num_args; i++) 
 	  if (strcmp(args[i].name, "height")) {
-		  child_args[i] = args[i];
+		  child_args[num_child_args] = args[i];
 	  	  num_child_args++;
 	  }
   /* Create the widgets that make up a ComboBox. */
